docs/source/10-minutes-to-dask.rst

```diff
@@ -373,6 +373,7 @@ triggering computation, we can inspect the task graph to figure out what's going
          >>> result.visualize()
 
       .. image:: images/10_minutes_dataframe_graph.png
+         :alt: Dask task graph for the Dask dataframe computation. The task graph shows a "loc" and "getitem" operations selecting a small section of the dataframe values, before applying a cumulative sum "cumsum" operation, then finally subtracting a value from the result. This computation can be written as "ddf["2021-10-01": "2021-10-09 5:00"].a.cumsum() - 100".
```

Comment:
- Should we use double backticks (or single backticks) instead of quotes around the code snippet at the very end?
- Does the last part fail to play nice with screen readers? (Hopefully not, otherwise it'd be hard to read the rest of the code on the webpage) If so, should we get rid of the equation altogether?

Note: I wrote this alt-text sentence, but I'm not sure about the answers to the questions above. Happy to have things changed.

Reply:
> Should we use double backticks (or single backticks) instead of quotes around the code snippet at the very end?

I couldn't find any recommendations about formatting code in alt text, but my instinct is that we shouldn't use backticks, which are more of an rst/markdown formatting option. My understanding is that screen readers are very focused on HTML, and I worry that other markup would be confusing. Since the alt text is just a string literal, I think human-readable quotes are okay.

> Does the last part fail to play nice with screen readers?

I think it would play okay with screen readers, but I might still advocate removing it, since it's already in the text of the document, and most of the recommendations around alt-text caution against adding redundant information.

Reply:
Sounds good. I've added github code suggestions to remove the redundant equations.